indico/core/db/sqlalchemy/util/models.py

```diff
@@ -30,9 +30,39 @@
 from indico.core import signals
 
 
+class IndicoBaseQuery(BaseQuery):
+    def paginate(self, page=1, per_page=20, show_all=False):
+        """ This function paginates the results while at the same time
```

Comment:
Very weird formatting. If the summary doesn't fit in one line the `"""` goes in its own line. Anyway, why not something concise?

```
"""Paginate a query object.

This behaves almost like the default `paginate` method from
Flask-SQLAlchemy but allows showing all results on a single page.
"""
```